### DISCUSSION TITLE
JLReq-d 編集会議 2024-10-31

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#71] [General]
# JLReq-d 編集会議 2024-10-31
オフライン @ 東京パークタワー

### 出席者
敏先生 @KobayashiToshi, 木田 @kidayasuo 

### ノート
- 前回の続き、2.3 文字の使い方をドラフト
- 2.4 日本語書体とフォントの「日本語書体の種類」までレビュー
- 変更を[チェックイン済み](https://github.com/w3c/jlreq-d/blob/gh-pages/drafts/2.%20日本語デジタルテキスト入門（Introduction%20to%20Japanese%20Digital%20Text）.md)
- 次回11/14 10:00からオンライン

----

**Comment by himorin** [comment]
リンクをブラウザのアドレスバーからコピーした%エスケープのもので張るとダブルエスケープされてしまうので表示できなくなるようです。
正しくは [チェックイン済み](https://github.com/w3c/jlreq-d/blob/gh-pages/drafts/2. 日本語デジタルテキスト入門（Introduction to Japanese Digital Text）.md) から。（多分これで行けるはず

**Reply by himorin** [reply]
あ、ダメだった。。半角空白入りURLを張る手段はなさそう？

**Reply by kidayasuo** [reply]
ありがとうございます。確かに壊れてますね……、調べた結果、普通にアドレスバーのものを素直に貼ると良いようです。最初のは、他のエディタからコピーする際に、二重にエスケープされてしまったんでしょうかね。直しました。
